src/main/java/racingCar/Cars.java

```diff
@@ -0,0 +1,59 @@
+package racingCar;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+	private final List<Car> cars=new ArrayList<>();
```

Comment:
아래와 같이  Cars의 cars 필드의 컬렉션을 내부에서 직접 생성하는 것이 아니라 `외부에서 주입`받도록해주세요. 
지금처럼 내부에서 ArrayList를 생성하여 cars 필드에 할당해주게되면 ArrayList에 강하게결합되게됩니다. 

외부에서 의존성을 주입받게되면 추후 런타임 시점에 유연하게 List의 하위구현체들을 무엇이든 생성자 파라미터로 주입받아 Cars 객체를 생성할 수 있기 때문에 조금 더 유연하게 객체를 생성하고 ArrayList가 아닌 다른 List 구현체들을 활용할 수 있게 됩니다. 

```suggestion
	private final List<Car> cars;

	public Cars(List<Car> cars) {
		this.cars = cars;
	}
	
```

아래의 글을 참고해보시면 조금 더 이해하시는데 도움이되실 것 같아요. 😃
[다형성이란?](https://tecoble.techcourse.co.kr/post/2020-10-27-polymorphism/)